src/utils/mod.rs

```diff
@@ -865,7 +867,7 @@ pub struct Signature {
 #[derive(Debug, PartialEq)]
 pub enum SignatureData {
     RsaSignature(Vec<u8>),
-    EccSignature { r: Vec<u8>, s: Vec<u8> },
+    EcdsaSignature { r: Vec<u8>, s: Vec<u8> },
```

Comment:
These seem to be remnants from a change I made (and since merged) that you might've merged into your branch